gnmvidispine-py36.spec

```diff
@@ -0,0 +1,47 @@
+%define name gnmvidispine-py36
+%define version 1.9
+%define unmangled_version 1.9
+%define release 1
+%define sourcebundle gnmvidispine-1.9.DEV.tar.gz
+
+Summary: An object-oriented Python interface to the Vidispine Media Asset Management system (Python 2.7)
+Name: %{name}
+Version: %{version}
+Release: %{release}
+Source0: gnmvidispine-1.9.DEV.tar.gz
+License: UNKNOWN
+Group: Development/Libraries
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-buildroot
+Prefix: %{_prefix}
+BuildArch: noarch
+Vendor: Andy Gallagher <andy.gallagher@theguardian.com>
+Url: https://github.com/fredex42/gnmvidispine
+AutoReqProv: no
+Requires: python(abi) = 3.6 python36-pytz python36-dateutil python36-future
```

Comment:
were there any other python3 rpms you needed to install here?

Reply:
No.